packages/rpc-methods/src/permitted/invokeSnapSugar.ts

```diff
@@ -8,6 +8,11 @@ import {
 import { isObject } from '@metamask/utils';
 import { SNAP_PREFIX } from '@metamask/snaps-utils';
 
+export type InvokeSnapSugarArgs = {
+  snapId: string;
+  request: string;
```

Comment:
`request` is a JSON-RPC request, not a string.

Reply:
Updated.